src/cmd.cpp

```diff
@@ -150,30 +201,6 @@ static int subcommand_dem2tintiles(bool need_help,
     if("zemlya" == meshing_method || "terra" == meshing_method)
     {
         max_error = input_raster->get_cell_size();
```

Comment:
It seems that this line will always erase a max_error value passed throught CLI

Reply:
yes, that was the case in the original code. So I guess that was on purpose? I cannot tell if that was intentional or not.

Reply:
ok, completely messed this up. fixed.